src/pages/Signup/SIgnup.js

```diff
@@ -2,7 +2,38 @@ import React from 'react';
 import './Signup.scss';
 
 const Signup = () => {
-  return <div className="signup">Signup</div>;
+  return (
+    <div className="signup">
+      <form className="signupForm">
+        <img src="images/nike.png" alt="logo" className="logoImage" />
+        <legend className="formTitle">이제 나이키 멤버가 되어볼까요?</legend>
+        <div className="signupContainter">
+          <input type="text" />
+          <div className="pwContainer">
+            <input type="password" className="pw" placeholder="비밀번호" />
+            <i class="fa-regular fa-eye" />
+          </div>
+          <div className="pwRules">
+            <div className="pwLength">
+              <i class="fa-solid fa-xmark" />
+              <span>최소 8자</span>
+            </div>
+            <div className="pwCondition">
+              <i class="fa-solid fa-xmark" />
+              <span>알파벳 대문자 및 소문자 조합, 최소 1개 이상의 숫자</span>
+            </div>
+          </div>
+          <input type="date" className="birth" />
+          <p className="birthRules">생일에 나이키 멤버 리워드를 받으세요.</p>
+          <label className="TermsAgree">
```

Comment:
```suggestion
          <label className="termsAgree">
```
convention 확인 해 주세요~